### ISSUE TITLE
Bug with doing pr checkout not working as expected

### ISSUE THREAD

**Issue by PaulHofma** [ing-bank/doing-cli#54]
Using a valid PR_ID, I used

`doing pr checkout PR_ID`

which returned the error:

```
        $ Running command: az repos pr checkout --id ('79276',)`
az repos pr checkout --id ('79276',)
/bin/sh: -c: line 0: syntax error near unexpected token `('
/bin/sh: -c: line 0: `az repos pr checkout --id ('79276',)'
```

Trying the plain `az` command in my terminal reveals that it looks like `az` is not expecting the parentheses nor the comma.

**Comment by timvink** [comment]
Thanks for reporting, will look into it!

**Comment by timvink** [comment]
Fixed & released in [v1.1.7](https://github.com/ing-bank/doing-cli/releases/tag/v1.1.7)
